ion/bitstream_test.go

```diff
@@ -109,3 +112,188 @@ func TestBitcodeString(t *testing.T) {
 		}
 	}
 }
+
+func TestBinaryReadTimestamp(t *testing.T) {
+	test := func(ion []byte, eval string) {
+		t.Run(eval, func(t *testing.T) {
+
+			b := bitstream{}
+			b.InitBytes(ion)
+			b.Next()
+
+			val, err := b.ReadTimestamp()
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			et, err := time.Parse(time.RFC3339Nano, eval)
```

Comment:
Similar comment here: if `et` is meant to mean `expectedTime`, please expand it.

Reply:
Updated to `expectedTime`